packages/lifecycle/src/runLifecycleHook.ts

```diff
@@ -48,6 +48,7 @@ export default async function runLifecycleHook (
     config: opts.rawConfig,
     dir: opts.rootModulesDir,
     extraBinPaths: opts.extraBinPaths || [],
+    extraEnv: { PNPM_SCRIPT_SRC_DIR: opts.pkgRoot },
```

Comment:
might be better to name it `PNPM_LIFECYCLE_DIR`

Reply:
PNPM_SCRIPT_SRC_DIR is quite clear.

Reply:
Yes, but there is an env variable called `npm_lifecycle_event`. So maybe it would be more consistent.